### ISSUE TITLE
Loss functions for pretrained model

### ISSUE THREAD

**Issue by Richard-Antwi22** [Esri/arcgis-python-api#1511]
Hi, I trained an object detection model based on YOLOv3 with a Darknet53 backbone in ArcGIS Pro. Does anybody know what loss functions or how to get the default loss functions used by the YOLOv3 model in ArcGIS Pro?

**Comment by kapil-varshney** [comment]
The loss functions used for the bbox regression are Binary Cross Entropy (BCE) and L2. The loss function for the classification of classes is BCE.

**Comment by priyankatuteja** [comment]
@Richard-Antwi22 Please feel free to reopen issue if your doubts are unanswered
